scene/open-mobile-scene-package/README.md

```diff
@@ -1,26 +1,30 @@
 # Open mobile scene package
 
-Open and display a scene from an offline Mobile Scene Package (.mspk).
+Opens and displays a scene from a Mobile Scene Package (.mspk).
```

Comment:
this can be sentence case as per usage elsewhere in Esri documentation
```suggestion
Opens and displays a scene from a mobile scene package (.mspk).
```